spec/unit/optmath_spec.lua

```diff
@@ -0,0 +1,49 @@
+describe("Math module", function()
+
+    setup(function()
+        require("commonrequire")
+        Math = require("optmath")
+    end)
+
+    it("should round away from zero", function()
+        local num = 1.5
```

Comment:
I think it can be
assert.are.same(2, Math.roundAwayFromZero(1.5))

Or are you trying to use the evaluation statement as the "line separator" of different cases?